Cargo.toml

```diff
@@ -15,15 +15,19 @@ description = "a small crate to work with NEAR token values ergonomically and ef
 [dependencies]
 serde = { version = "1", features = ["derive"], optional = true }
 borsh = { version = "1", features = ["derive"], optional = true }
-schemars = { version = "0.8.8", optional = true }
+schemars-v1 = { version = "1.0.3", optional = true, package = "schemars" }
+schemars-v0_8 = { version = "0.8.8", optional = true, package = "schemars" }
 interactive-clap = { version = ">=0.2,<0.4", optional = true }
 
 [dev-dependencies]
 serde_json = { version = "1" }
 
 [features]
+__all-non-conflicting-features = ["serde", "interactive-clap", "borsh"]
 abi = ["borsh/unstable__schema", "schemars"]
```

Comment:
There are no action items here, but I spotted a couple of issues:

1. We should have named "abi" as "near-abi" to be more specific as "near-token" + "abi" is not really clear which ABI we are even talking about
2. It is going to be another pain point to migrate from "schemars" for "abi" feature as it is technically a breaking change

Reply:
It's gonna be a pain point anyway when we migrate to schemars v1 for abi, no matter the current change, right?